e2e/transform/transform-snapshotResolver/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "esModuleInterop": true
```

Comment:
Maybe avoid that given https://github.com/facebook/jest/pull/8842?
Not that it matters for the goal of that PR, but for consistency in the code base